org.opent2t.sample.hub.superpopular/com.wink.hub/js/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "opent2t-wink-hub",
+  "version": "1.0.0",
+  "description": "Hub schema and helper modules for Open Translators to Things",
+  "author": "OpenT2T",
+  "license": "MIT",
+  "main": "thingTranslator.js",
+  "files": [
+    "thingTranslator.js",
+    "package.json"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "githttps://github.com/openT2T/translators.git"
+  },
+  "bugs": {
+    "url": "https://github.com/openT2T/translators/issues"
+  },
+  "homepage": "https://github.com/openT2T/translators#readme",
+  "dependencies": {
+    "request": "^2.74.0",
+    "request-promise": "^4.1.1"
+  },
+  "devDependencies": {
+    "opent2t": "*",
+    "ava": "^0.15.2"
```

Comment:
This should be in dependencies, not devDependencies, because the schema has a runtime dependency on the schema reader in this package. Also specify a version, not just "*".